swesmith/bug_gen/llm/modify.py

```diff
@@ -48,7 +48,7 @@
 from typing import Any
 
 
-load_dotenv(dotenv_path=os.getenv("SWEFT_DOTENV_PATH"))
+load_dotenv()
```

Comment:
Removed the `os.getenv` for path override, because we don't do that anywhere else, so probably was never used